sdk/resources-subscriptions/arm-resources-subscriptions/CHANGELOG.md

```diff
@@ -1,15 +1,43 @@
 # Release History
+    
+## 3.0.0 (2023-05-08)
+    
+**Features**
 
-## 2.0.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-
+  - Added operation group Operations
+  - Added operation Subscriptions.checkZonePeers
+  - Added Interface AvailabilityZoneMappings
+  - Added Interface AvailabilityZonePeers
+  - Added Interface CheckZonePeersRequest
+  - Added Interface CheckZonePeersResult
+  - Added Interface ErrorDetail
+  - Added Interface ErrorResponseAutoGenerated
+  - Added Interface OperationAutoGenerated
+  - Added Interface OperationDisplayAutoGenerated
+  - Added Interface OperationListResultAutoGenerated
+  - Added Interface OperationsListNextOptionalParams
+  - Added Interface OperationsListOptionalParams
+  - Added Interface Peers
+  - Added Interface SubscriptionsCheckZonePeersOptionalParams
+  - Added Type Alias ActionType
+  - Added Type Alias OperationsListNextResponse
+  - Added Type Alias OperationsListResponse
+  - Added Type Alias Origin
+  - Added Type Alias SubscriptionsCheckZonePeersResponse
+  - Interface Location_2 has a new optional parameter availabilityZoneMappings
+  - Interface LocationMetadata has a new optional parameter geography
+  - Interface Operation has a new optional parameter actionType
+  - Interface Operation has a new optional parameter isDataAction
+  - Interface Operation has a new optional parameter origin
+  - Added Enum KnownActionType
+  - Added Enum KnownOrigin
+
+**Breaking Changes**
```

Comment:
Need to check the change is reasonable or not

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/23292#discussion_r1185657690 still in checking